cookbook/external_completers.md

```diff
@@ -46,8 +46,8 @@ This completer is not usable for allmost every other command, so it's recommende
 
 ```nu
 {
-    z: $zoxide_completer
-    zi: $zoxide_completer
+    z => $zoxide_completer
+    zi => $zoxide_completer
```

Comment:
Mhh this is not valid Nu code outside of a `match`. There may be some work necessary to get this section up to date. Previously, before `match` existed things like this where handled with records for which this is the correct syntax.

Reply:
I'm still a bit lost with everything. Just reading the docs it says that "This completer is not usable for allmost every other command, so it's recommended to add it as an override in the [multiple completer](https://www.nushell.sh/cookbook/external_completers.html#multiple-completer):". I copy-pasted it to the appropriate place and got the syntax error.

Reply:
No wonder it doesn't work, those snippets where written when the completer options were a `record | get | default` instead of a `match` :sweat_smile: 

As far as I can tell these changes should be enough, but maybe it should be made clearer where this configuration goes, and emphasize that it's a match statement?

Or just writing it as

```shell
match $spans.0 {
    ...
   __zoxide_z | __zoxide_zi => $zoxide_completer
    ...
}
```

Btw, found a typo at `This completer is not usable for alLmost ...` 

Happy to lend a hand rewriting this if needed :) As far as I can see no other snippets need updating

Reply:
I fixed the typo and added zoxide to the full example, I hope this is what you had in mind.